src/pages/commits/ActiveCommits/buildVariantCard/FailedTaskStatusIcon.tsx

```diff
@@ -0,0 +1,84 @@
+import { useLazyQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import IconButton from "@leafygreen-ui/icon-button";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Body } from "@leafygreen-ui/typography";
+import { Skeleton } from "antd";
+import { Link } from "react-router-dom";
+import { TaskStatusIcon } from "components/TaskStatusIcon";
+import { getTaskRoute } from "constants/routes";
+import {
+  GetFailedTaskStatusIconTooltipQuery,
+  GetFailedTaskStatusIconTooltipQueryVariables,
+} from "gql/generated/types";
+import { GET_FAILED_TASK_STATUS_ICON_TOOLTIP } from "gql/queries";
+import { msToDuration } from "utils/string";
+
+interface FailedStatusIconProps {
+  taskId: string;
+  status: string;
+}
+export const FailedTaskStatusIcon: React.FC<FailedStatusIconProps> = ({
+  taskId,
+  status,
+}) => {
+  const [loadData, { data, loading }] = useLazyQuery<
+    GetFailedTaskStatusIconTooltipQuery,
+    GetFailedTaskStatusIconTooltipQueryVariables
+  >(GET_FAILED_TASK_STATUS_ICON_TOOLTIP, { variables: { taskId } });
```

Comment:
We should only attempt to fetch test results if we confirm this component has a failed status.